### ISSUE TITLE
Version 2.3.2: Wrong translation of Vision datapoint  xxx.mower.partyModus to cfg.sc.enabled

### ISSUE THREAD

**Issue by hsteinme** [iobroker-community-adapters/ioBroker.worx#761]
The Vision datapoint  xxx.mower.partyModus does not have a correct effect on the mower. Reason: partyModus = true is incorrectly translated to cfg.sc.enabled = 1 instead of cfg.sc.enabled = 0.

```
 sendPartyModus(id, value, mower) {
        if (mower.capabilities != null && mower.capabilities.includes("vision")) {
            const val = value ? 1 : 0;
            this.sendMessage(`{"sc":{ "enabled":${val}}}`, mower.serial_number, id);
        } ...
```
Problem was originally reported in the [roboter forum](https://www.roboter-forum.com/threads/iobroker-anbindung-m600.65277/),

**Comment by Lucky-ESA** [comment]
@hsteinme 
Thanks for this info. Is fixed with next release.
BR//Lucly

**Comment by EngelenJ** [comment]
Nice, thank you!

**Comment by Lucky-ESA** [comment]
Fixed in release 2.3.3 latest or stable >= 2.3.3
